Gems/GraphCanvas/Code/StaticLib/GraphCanvas/Widgets/NodePalette/TreeItems/NodePaletteTreeItem.cpp

```diff
@@ -88,7 +91,7 @@ namespace GraphCanvas
             case Qt::DecorationRole:
                 if (HasError())
                 {
-                    return m_errorIcon;
+                    return QIcon(":/GraphCanvasEditorResources/toast_error_icon.png");
```

Comment:
Awesome fix! Consider caching this somewhere global to avoid calling QIcon's weirdly slow constructor at all if you ever see this show up in a profile again.

Reply:
Good call! And yeah I want to dig further to see why the constructor is so slow too